sdk/lib/drivers/wdf/kmdf/src/dynamic/version/version.cpp

```diff
@@ -485,7 +497,7 @@ WDF_LIBRARY_REGISTER_CLIENT(
         rawData[3] = WdfLibraryInfo.Version.Minor;
 
         LibraryLogEvent(FxLibraryGlobals.DriverObject,
-                       WDFVER_MINOR_VERSION_NOT_SUPPORTED,
+                       0xdeadbeef, //WDFVER_MINOR_VERSION_NOT_SUPPORTED,
```

Comment:
I forgot why I added this :)
There is no `WDFVER_MINOR_VERSION_NOT_SUPPORTED` defined?

Reply:
No it's not defined